web/src/components/Error.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react"
 import { ListGroup, Table, Button, Badge } from "react-bootstrap"
-import * as moment from "moment"
+import moment from "moment"
```

Comment:
as a side-note: momentjs is no longer maintained, we should plan to move away from it